src/kudu/consensus/consensus.proto

```diff
@@ -542,6 +542,14 @@ message RunLeaderElectionRequestPB {
 
   // the id of the tablet
   required bytes tablet_id = 1;
+
+  // Time when the original server was promoted away from. We can use this to
+  // measure the total time of a chain of promotions
+  optional fixed64 original_start_time = 3;
```

Comment:
why fixed64 and what does this integer represent? UTC?

Reply:
I followed how other timestamps are stored here, for example like in line 482. It's currently used below as nanoseconds since epoch.